e2e-tests/run-release.csv

```diff
@@ -13,6 +13,7 @@ haproxy
 init-deploy
 limits
 monitoring-2-0
+monitoring-pmm3,8.0
```

Comment:
Why we need version here?

Reply:
similar comment with here: https://github.com/percona/percona-xtradb-cluster-operator/pull/2056#discussion_r2069312510